tests/ast/data/col_alias.test

```diff
@@ -22,7 +22,7 @@ df = df.select(col("A").alias("test"))
 
 df = df.select(col("A").name("test"))
 
-df = df.select((col("A") + 1).name("test"))
+df = df.select(col("A") + 1.name("test"))
```

Comment:
this looks wrong.

Reply:
I am working on that one atm... Seems some broken unparser code